crates/lib/src/db/raw_def/v9.rs

```diff
@@ -296,6 +296,37 @@ pub struct RawScheduleDefV9 {
     pub reducer_name: RawIdentifier,
 }
 
+/// A constraint definition attached to a table.
+#[derive(Debug, Clone, SpacetimeType)]
+#[sats(crate = crate)]
+#[cfg_attr(feature = "test", derive(PartialEq, Eq, PartialOrd, Ord))]
+pub struct RawConstraintDefV9 {
+    /// The name of the constraint. Must be unique within the containing `RawDatabaseDef`.
```

Comment:
I see this type (`RawDatabaseDef`) mentioned in a lot of comments in `def.rs` and `v9.rs`, but it doesn't seem defined anywhere?

Reply:
Yep, it became `RawModuleDef`. Good catch.